companion/src/modeledit/telemetry.h

```diff
@@ -82,6 +83,7 @@ class TelemetryCustomScreen: public ModelPanel
     void barSourceChanged(int index);
     void barMinChanged(double value);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1144)